pkg/cmd/server/origin/auth.go

```diff
@@ -375,8 +389,17 @@ func (c *AuthConfig) getAuthenticationHandler(mux cmdutil.Mux, errorHandler hand
 				}
 				passwordSuccessHandler := handlers.AuthenticationSuccessHandlers{c.SessionAuth, redirectSuccessHandler{}}
 
+				loginPath := OpenShiftLoginPrefix
```

Comment:
go ahead and make `OpenShiftLoginPrefix` private... don't want other parts of the code base assuming that is the full login path now that we sometimes suffix it